runtime/server_http_request.go

```diff
@@ -651,6 +651,20 @@ func (req *ServerHTTPRequest) UnmarshalBody(
 	return true
 }
 
+// WriteBytesBody replace the request body
```

Comment:
"ReplaceBodyWith replaces the raw request body with given body and updates the request  content-length header accordingly. 
This method is only supposed to be used in middlewares where request body needs to be modified. And the encoding of the body should stay the same."

Reply:
Looks god. Will change.